config/settings.yml

```diff
@@ -7,6 +7,7 @@ servers:
   sample_config_path: 'config/sample.server.ovpn.erb'
 
 mailchimp:
+  subscription: 'true'
```

Comment:
I'd prefer to manage mailchimp through env variables.
For example we can just check that api key exists.
So, basically we can check just `return unless Settings.mailchimp.api_key`

Reply:
@reabiliti ping